gradle-incremental-configuration-cache/src/test/groovy/com/palantir/gradle/configcache/incremental/IncrementalConfigurationCacheTest.groovy

```diff
@@ -89,7 +101,25 @@ class IncrementalConfigurationCacheTest extends IntegrationTestKitSpec {
         !output.contains('Configuration cache entry stored.')
     }
 
-    private boolean runTasksWithConfigurationCache(String... tasks) {
+    @Unroll
+    def "reassure users iff there are configuration cache problems"() {
+        file("gradle/configuration-cache-allowed-tasks") << ""
+
+        when:
+        def buildResult = createRunner([task, '--configuration-cache'] as String[]).build()
+
+        then:
+        buildResult.output.contains(
```

Comment:
Best to save just the output in the `def` from the `when` for better test errors.

Reply:
oops sorry i missed this! resolved.